modules/concertBidAdapter.js

```diff
@@ -47,10 +46,18 @@ export const spec = {
         optedOut: hasOptedOutOfPersonalization(),
         adapterVersion: '1.1.1',
         uspConsent: bidderRequest.uspConsent,
-        gdprConsent: bidderRequest.gdprConsent
+        gdprConsent: bidderRequest.gdprConsent,
+        gppConsent: bidderRequest.gppConsent,
       }
     };
 
+    if (!payload.meta.gppConsent && bidderRequest.ortb2?.regs?.gpp) {
+      payload.meta.gppConsent = {
+        gppString: bidderRequest.ortb2.regs.gpp,
+        applicableSections: bidderRequest.ortb2.regs.gpp_sid
```

Comment:
(For the record, I made a comment here, then realised it was wrong (still getting caffeinated), so deleted it. But you'll have gotten the email, so I wanted to follow-up.)